widget/playground/src/containers/StyleLayout/StyleLayout.Preset.tsx

```diff
@@ -0,0 +1,160 @@
+import type { ColorsTypes, PresetTypes } from './StyleLayout.types';
+import type { WidgetColors } from '@rango-dev/widget-embedded';
+
+import { Button, Divider, Switch, Typography } from '@rango-dev/ui';
+import React, { useState } from 'react';
+
+import { CUSTOM_THEMES } from '../../constants';
+import { useConfigStore } from '../../store/config';
+import { shallowEqual } from '../../utils/common';
+
+import {
+  Collapse,
+  Color,
+  ColorsContent,
+  PresetContent,
+  PresetTheme,
+  Row,
+} from './StyleLayout.styles';
+
+const Colors = (props: ColorsTypes) => {
+  return (
+    <ColorsContent>
+      <Color style={{ backgroundColor: props.primary }} />
+      <Color
+        style={{
+          backgroundColor: props.secondary,
+          position: 'absolute',
+        }}
+      />
+      <Color
+        style={{
+          backgroundColor: props.background,
+          position: 'relative',
+        }}
+      />
+    </ColorsContent>
+  );
+};
+
+const FIX_HEIGHT = 41;
+const NUMBER_ITEMS_2_COL = 4;
+const NUMBER_ITEMS_1_COL = 2;
+
+export function Preset(props: PresetTypes) {
+  const { tab } = props;
+  const [showMore, setShowMore] = useState(true);
+  const onChangeTheme = useConfigStore.use.onChangeTheme();
+  const { theme } = useConfigStore.use.config();
+  const onSelectTheme = useConfigStore.use.onSelectTheme();
+  const CUSTOM_THEMES_FILTER = CUSTOM_THEMES.filter(
+    (theme) =>
+      ((tab === 'dark' || tab === 'light') && Object.keys(theme[tab]).length) ||
+      (tab === 'auto' &&
+        Object.keys(theme.dark).length &&
+        Object.keys(theme.light).length)
+  );
```

Comment:
We could rewrite this code like this to make it more readable.
Please also reuse isAutoTab, ... in next lines.
```
const isDarkTab = tab === 'dark';
const isLightTab = tab === 'light';
const isAutoTab = tab === 'auto';

const CUSTOM_THEMES_FILTER = CUSTOM_THEMES.filter((theme) => {
  const hasDarkKeys = Object.keys(theme.dark).length > 0;
  const hasLightKeys = Object.keys(theme.light).length > 0;

  return (
    (isDarkTab || isLightTab) && (hasDarkKeys || hasLightKeys) ||
    (isAutoTab && hasDarkKeys && hasLightKeys)
  );
});
```

Reply:
fixed